kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h

```diff
@@ -1068,26 +1061,25 @@ class ResidualBasedBlockBuilderAndSolver
             }
         }
 
-        #pragma omp parallel for firstprivate(system_size)
-        for (int k = 0; k < static_cast<int>(system_size); ++k) {
-            std::size_t col_begin = Arow_indices[k];
-            std::size_t col_end = Arow_indices[k+1];
-            const double k_factor = scaling_factors[k];
+        IndexPartition<int >(static_cast<int>(system_size)).for_each([&](int Index){

```

Comment:
please use `std::size_t`
`int` was only used bcs of OpenMP

This applies generally

Reply:
Yes, but it was giving an error on line 1071 `if (static_cast<int>(Acol_indices[j]) != Index )`. 
Will remove that Typecasted `Acol_indices[j]` and will change accordingly. 
Thanks :)